docs/manuals/gui/viselements/controls.md_template

```diff
@@ -1,10 +1,24 @@
 # Controls
 
-Controls are user interface objects that represent data.
+Controls are user interface objects that represent data and that users can interact with.
 
-## Controls list
+## Generic controls
 
-Here is the list of all available controls in Taipy:
+Here is the list of all available controls in Taipy GUI:
```

Comment:
Gui is still in?

Reply:
What about removing GUI here and changing the section title to "Gui controls"?

Reply:
"Gui controls" to me sounds very awkward. There can be no controls without a GUI.
The term "Generic" indicates that these are available no matter what. That is, having taipy-gui installed.
Let's discuss this further. I'm not really happy either.

Reply:
I rephrased that a little bit, using the "Standard controls" term for the ones that specifically are in taipy-gui.
Also added a note that explains that (as soon as taipy-gui is installed, you're good to go).